en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -8,7 +8,7 @@ Not all languages allow a 1:1 translation of the concepts we have included in th
 
 ## Experimenting with JavaScript
 
-You are going to be experimenting with JavaScript, so you will need a quick way of inspecting expressions and obtaining feedback on what you are doing. The best way is to use the **Node console.** For instance:

```

Comment:
One of the first things I asked Salva when I started working on the text was whether we'd be going for UK or US English. I was told to go with US, and that's the style I've tried to keep for the whole text in terms of spelling (to which degree of success, that's a question I'd prefer to leave to proofreaders). I had a quick review of my textbooks and some style guides, and most American style guides seem to agree upon leaving punctuation inside quotes & direct speech, typographical emphasis, parentheses and other such cases (a notable exception would be hypertext links, on which most if not all recommendations are against including punctuation, even when they are not explicit URLs; I also made an exception for code-styled language within the text). Abhorrent as I personally find the practice, I decided to adopt it for the sake of consistency. However, since both you and Salva have raised this objection and I'm sure there will be more opinions like yours, I'm not going to waste any more time trying to defend the indefensible :P